monitor_vitals.py

```diff
@@ -0,0 +1,43 @@
+from time import sleep
+import sys
+
+class Monitor_Vitals:
+  def is_temperature_ok(self, temperature):
+    if temperature in range(95, 102):
+      self.print_loading_message()
```

Comment:
The purpose of showing the delay is to grab the user's attention when something is abnormal.
It isn't needed when things are normal.